deploy/olm-catalog/compliance-operator/manifests/compliance.openshift.io_tailoredprofiles_crd.yaml

```diff
@@ -83,6 +83,25 @@ spec:
               extends:
                 description: Points to the name of the profile to extend
                 type: string
+              manualRules:
```

Comment:
Does adding this type of stuff to the CRD impact the versioning, or is it safe to include it in a patch version since it's an additional backwards compatible change and the APIs are still alpha?

Reply:
Since this is optional, I don't think it will have backward compatibility issues